src/IIIFPresentation/API.Tests/Integration/GetManifestTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿#nullable disable
+
+using System.Net;
+using API.Tests.Integration.Infrastructure;
+using Core.Response;
+using IIIF.Presentation.V3;
+using Test.Helpers.Integration;
+
+namespace API.Tests.Integration;
+
+[Trait("Category", "Integration")]
+[Collection(CollectionDefinitions.StorageCollection.CollectionName)]
+public class GetManifestTests : IClassFixture<PresentationAppFactory<Program>>
```

Comment:
Do these need fleshed out?
* Flat should contain the custom properties (`slug`, `parent` etc)
* Requests that end in 404 / 401 / 303